en/README.md

```diff
@@ -0,0 +1,1138 @@
+        "groupId": "1"
+      }
+    ]
+  },
+  "method": "putHostGroups",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putHostGroupMembership(method)
+
+Send all host group membership information to Hatohol server to use "ALL" option when complate conect to Hatohol server and changed hosts information that saved in internal.
+
+ In case of using "UPDATE" option, send diffarence to use lastInfo that got using [getLastInfo](#user-content-getlastinfomethod) or saved in internal to Hatohol server.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostGroupMembership|object array|M|-|This is object array that stores host group membership information. Look at the following table for details.|
+|updateType|string    |M|-|Select sending option that match the situation from [[List](#user-content-updatetype)].|
+|lastInfo    |String32767 |O|-|Insert to here host group membership information of last. This information is result of [getLastInfo](#user-content-getlastinfomethod).|
+
+***hostGroupMembership object***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostId  |String255    |M|-|Id of host.|
+|groupIds|String255 array|M|-|Id of host group.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "updateType": "ALL",
+    "lastInfo": "201504091056",
+    "hostGroupMembership": [
+      {
+        "groupIds": [
+          "1",
+          "2",
+          "5"
+        ],
+        "hostId": "1"
+      }
+    ]
+  },
+  "method": "putHostGroupMembership",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putTriggers(method)
+
+Send all triggers information to Hatohol server to use "ALL" option when complate conect to Hatohol server or reseive [fetchTriggers](#user-content-fetchtriggersmethod) procedure from Hatohol server.
+
+ In case of using "UPDATE" option, send diffarence to use lastInfo that got using [getLastInfo](#user-content-getlastinfomethod) or saved in internal to Hatohol server.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|triggers    |object array|M|-|This is object array that stores triggers information. Look at the following table for details.|
+|updateType|string    |M|-|Select sending option that match the situation from [[List](#user-content-updatetype)].|
+|lastInfo    |String32767|O|-|Insert to here trigger information of last. This information is result of [getLastInfo](#user-content-getlastinfomethod).|
```

Comment:
> This information is result of getLastInfo.

A result of [getLastInfo](#user-content-getlastinfomethod) is constructed from this information.